src/components/AddPlaidBankAccount.js

```diff
@@ -69,6 +72,7 @@ const defaultProps = {
     plaidLinkOAuthToken: '',
     allowDebit: false,
     bankAccountID: 0,
+    selectedPlaidIndex: undefined,
```

Comment:
Why set it to undefined instead of null? 🤔

Reply:
Done